### DISCUSSION TITLE
ESP Terminal with 3.5inch RGB Capacitive Touch Display

### DISCUSSION THREAD

**Discussion by gromgsxr** [Bodmer/TFT_eSPI#2812] [Q&A - General]
im trying to get the ESP Terminal with 3.5inch RGB Capacitive Touch Display working its a 16bit parallel display i was trying in 8 bit mode but can only get it to display a white screen 

https://www.elecrow.com/wiki/index.php?title=ESP_Terminal_with_3.5inch_RGB_Capacitive_Touch_Display

i have tried editing one of the user_setup files to match the pinout provided in there wiki and verified its using these with the read user setup sketch but all i get is a white screen

**Comment by Bodmer** [comment]
The display requires a 16 bit parallel interface and unfortunately TFT_eSPI does not support that.
